packages/theme-check-common/src/checks/index.ts

```diff
@@ -35,6 +34,8 @@ import { ValidJSON } from './valid-json';
 import { ValidSchema } from './valid-schema';
 import { ValidStaticBlockType } from './valid-static-block-type';
 import { VariableName } from './variable-name';
+import { ValidSchemaName } from './valid-schema-name';
+import { UniqueStaticBlockId } from './unique-static-block-id';
```

Comment:
Can you use the `Organize imports` vscode command to make sure these are sorted? IIRC the default keyboard shortcut is `ctrl+shift+o`

We have #540 to make this automatic, but never got around doing it.